src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { TabsType } from '../../types/Tabs';
+
+export const Tabs: React.FC<TabsType> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentСomponent = tabs.filter(tab => tab.id === selectedTabId);
```

Comment:
Same here. Also, why do you use a filter here instead of find?
Only one tab can be selected at a time